### DISCUSSION TITLE
numeric input  inc/dec  buttons

### DISCUSSION THREAD

**Discussion by nedimkeskin** [primefaces/community#2915] [PrimeNG]
When I put inc/dec buttons in the numeric input field (p-inputnumber) , the buttons do not appear as in the examples and are not placed in the input field.

I did all the suggested style forcing but it doesn't work.
![correct](https://github.com/user-attachments/assets/f8602feb-2085-4b99-abf1-64529aed0add)
example 
![incorrect](https://github.com/user-attachments/assets/0250a2ce-8a2e-4037-8e0b-3b51b9a83ccd)
my input

**Comment by qwadrox** [comment]
Which Primeng version do you have and what theme are you using? 
Could you make a stackblitz reproduction of this behavior?

**Comment by nedimkeskin** [comment]
After much effort, I noticed that, When i  do the following remark in index.html, it works correctly.
remark  this line    link id="theme-css" rel="stylesheet" type="text/css" href="assets\theme\bootstrap4-dark-blue"
But I want to change the theme in runtime.
